src/components/ClipboardButton/FakeButton.tsx

```diff
@@ -0,0 +1,40 @@
+import React, {MouseEventHandler} from 'react';
+import {block} from '../utils/cn';
+import {useButtonHandlers} from '../utils/useButtonHandlers';
+import {ClipboardIcon} from '../ClipboardIcon';
+import {CopyToClipboardStatus} from '../CopyToClipboard';
+import {ClipboardButtonProps} from './ClipboardButton';
+import './ClipboardButton.scss';
+
+const b = block('clipboard-button');
+
+const DEFAULT_ICON_SIZE = 24;
+
+interface FakeButtonProps extends Pick<ClipboardButtonProps, 'className' | 'size'> {
+    status: CopyToClipboardStatus;
+    'data-qa': ClipboardButtonProps['qa'];
+    onClick?: MouseEventHandler<HTMLSpanElement>;
+}
+
+export function FakeButton({
```

Comment:
What is the point of extracting it into separate component if we proxy all props? The original component wasn't so big

Reply:
I need to call hook inside, that adds keyboard handlers. onClick here is passed via cloning node, that returned from clipboard children callback